### ISSUE TITLE
[Feature Request]: SystemMenu to be theme related

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Standard-Toolkit#648]
Ad part of the work on #646 there is a handler for the system menu now.

This could be replaced by a KryptonContextMenu with custom controls:
See the following locations for some guidance:

https://www.actiprosoftware.com/blog/post/25194/wpf-windowchrome-system-menu-customization
https://stackoverflow.com/questions/22852461/win32-api-getmenuiteminfo-returns-only-the-first-character-of-the-item-text
https://www.codeguru.com/csharp/manipulating-the-system-menu-using-c/

Some indicators of what to look for in the "Actions" / fields / bitmaps / enablement of menus here:
https://stackoverflow.com/questions/49056495/how-to-show-menu-item-with-icon-and-text-in-winforms-system-menu/49065061?noredirect=1#comment85223866_49065061

**Comment by PWagner1** [comment]
https://github.com/user-attachments/assets/1ddfb617-4faf-4543-9e0c-2d912db04c5e

**Comment by giduac** [comment]
@PWagner1 

V85 cannot build testform.
V100 cannot use the designer.
Both after the merge of 648.

As discussed in
https://discord.com/channels/1261930150585569350/1406345107816579203

**Comment by giduac** [comment]
@PWagner1 

In V85 and V95 these debug message flood the output window. `Image Info - Type: Restore`
Can these please be taken out?

**Comment by PWagner1** [comment]
> [@PWagner1](https://github.com/PWagner1)
> 
> In V85 and V95 these debug message flood the output window. `Image Info - Type: Restore` Can these please be taken out?

Will do

**Comment by PWagner1** [comment]
Reopening for re-investigation in V110

**Comment by PWagner1** [comment]
@giduac 

Can this be closed, or is there still more work to be done?

**Comment by giduac** [comment]
> [@giduac](https://github.com/giduac)
> 
> Can this be closed, or is there still more work to be done?

Nah, there's another bug in it which I made fix for but will be posted after v100 is online, next week somewhere..
